wsmaster/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/wsagent/WsAgentLauncherImpl.java

```diff
@@ -105,13 +105,20 @@ public void startWsAgent(String workspaceId) throws NotFoundException, MachineEx
         } catch (BadRequestException wsAgentLaunchingExc) {
             throw new MachineException(wsAgentLaunchingExc.getLocalizedMessage(), wsAgentLaunchingExc);
         }
+        LOG.error(String.format("Fail pinging ws agent. Workspace ID:%s. Url:%s. Timestamp:{}",workspaceId,UriBuilder.fromUri(devMachine.getRuntime()
```

Comment:
Please do not use String.format here. LOG.error has its own API to submit parameters - "{}"